### PR TITLE
fix(developer): kmc: fix jis/abnt2 scan codes 🙀

### PR DESCRIPTION
**Pull request by srl295**
for #8552

@keymanapp-test-bot skip

### CODE REVIEW

File: common/web/types/src/consts/virtual-key-constants.ts [hunk 1]
```diff
@@ -163,13 +170,13 @@ export const ISOVirtualKeyMap: KeyMap = [
 
 export const JISVirtualKeyMap: KeyMap = [
   // [Hankaku/Zenkaku] 1 2 3 4 5 6 7 8 9 0 - ^ ¥ [bksp]
-  [ k.K_1, k.K_2, k.K_3, k.K_4, k.K_5, k.K_6, k.K_7, k.K_8, k.K_9, k.K_0, k.K_HYPHEN, k.K_EQUAL, k.K_oE2 /* YEN? */ ],
+  [ k.K_1, k.K_2, k.K_3, k.K_4, k.K_5, k.K_6, k.K_7, k.K_8, k.K_9, k.K_0, k.K_HYPHEN, k.K_EQUAL, k.K_BKSLASH /* YEN */ ],
   // [tab] Q W E R T Y U I O P @«`» [ [enter]
   [ k.K_Q, k.K_W, k.K_E, k.K_R, k.K_T, k.K_Y, k.K_U, k.K_I, k.K_O, k.K_P, k.K_BKQUOTE, k.K_LBRKT  ],
   // [caps] A S D F G H J K L ; : ] [enter]
-  [ k.K_A, k.K_S, k.K_D, k.K_F, k.K_G, k.K_H, k.K_J, k.K_K, k.K_L, k.K_COLON, k.K_QUOTE, k.K_BKSLASH ],
+  [ k.K_A, k.K_S, k.K_D, k.K_F, k.K_G, k.K_H, k.K_J, k.K_K, k.K_L, k.K_COLON, k.K_QUOTE, k.K_RBRKT ],
```
Comment (srl295) [inline common/web/types/src/consts/virtual-key-constants.ts:177]: K_QUOTE here is actually the `:` and `*`|`せ` key.

File: common/web/types/src/consts/virtual-key-constants.ts [hunk 2]
```diff
@@ -163,13 +170,13 @@ export const ISOVirtualKeyMap: KeyMap = [
...
   // [shift] Z X C V B N M , . / _ [shift]
-  [ k.K_Z, k.K_X, k.K_C, k.K_V, k.K_B, k.K_N, k.K_M, k.K_COMMA, k.K_PERIOD, k.K_SLASH, k.k_oC1 /* ろ */ ],
+  [ k.K_Z, k.K_X, k.K_C, k.K_V, k.K_B, k.K_N, k.K_M, k.K_COMMA, k.K_PERIOD, k.K_SLASH, k.K_oE2 /* ろ */ ],
```
Comment (srl295) [inline common/web/types/src/consts/virtual-key-constants.ts:179]: per https://github.com/keymanapp/keyman/pull/8513#discussion_r1151741205

File: common/web/types/src/consts/virtual-key-constants.ts [hunk 3]
```diff
@@ -163,13 +170,13 @@ export const ISOVirtualKeyMap: KeyMap = [
 
 export const JISVirtualKeyMap: KeyMap = [
   // [Hankaku/Zenkaku] 1 2 3 4 5 6 7 8 9 0 - ^ ¥ [bksp]
-  [ k.K_1, k.K_2, k.K_3, k.K_4, k.K_5, k.K_6, k.K_7, k.K_8, k.K_9, k.K_0, k.K_HYPHEN, k.K_EQUAL, k.K_oE2 /* YEN? */ ],
+  [ k.K_1, k.K_2, k.K_3, k.K_4, k.K_5, k.K_6, k.K_7, k.K_8, k.K_9, k.K_0, k.K_HYPHEN, k.K_EQUAL, k.K_BKSLASH /* YEN */ ],
```
Comment (srl295) [inline common/web/types/src/consts/virtual-key-constants.ts:173]: per https://github.com/keymanapp/keyman/pull/8513#discussion_r1151741205

**Comment by srl295** [comment]
double checked ABNT2 per https://kbdlayout.info/kbdbr/overview+virtualkeys by @miloush (thanks!)

**Review by mcdurdin — APPROVED** [review]
LGTM I think!

**Review by mcdurdin — APPROVED** [review]
LGTM AFAICT

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.95-alpha](https://keyman.com/downloads/releases/alpha/17.0.95)

**Pull request merged**